### PR TITLE
Support MySQL integration

### PR DESCRIPTION
**Pull request by galiacheng**
Changes include:

- Add MySQL as an option in UI -> DB blade.
   - Show MySQL when using pathed images.
   - Show MySQL when using GA WLS 14 JDK11 image.
- Use env variables to pass parameters to deployment script in DB template.

Note: this PR will leverage the built-in JDBC driver, see:

-  WLS 14c [Using JDBC Drivers with WebLogic Server (oracle.com)](https://docs.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/jdbca/third_party_drivers.html): mysql-connector-java-commercial-8.0.14-bin.jar
- WLS 12214 [Configure Database Connectivity (oracle.com)](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/jdbca/configure-database-connectivity.html#GUID-2BAFE654-B10B-4A72-9D1C-A10FCCFBA458): mysql-connector-java-commercial-8.0.14-bin.jar
- WLS 12213 [Using JDBC Drivers with WebLogic Server (oracle.com)](https://docs.oracle.com/middleware/12213/wls/JDBCA/third_party_drivers.htm#JDBCA231): mysql-connector-java-commercial-5.1.22-bin.jar

Test offer: https://portal.azure.com/#create/microsoft_javaeeonazure_test.20210525-arm-oraclelinux-wls-on-aks-preview20210525-arm-oraclelinux-wls-on-aks

### CODE REVIEW

**Comment by galiacheng** [comment]
Working with SSL version issue on GA images

File: weblogic-azure-aks/src/main/arm/scripts/setupDBConnections.sh [hunk 1]
```diff
@@ -3,112 +3,75 @@
...
-    <databaseType>
-    <dbUser>
-    <dsConnectionURL>
-    <jdbcDataSourceName>
-    <wlsDomainUID>
-    <wlsUser>
-    <dbOptType>
-END
-)
-    echo_stdout "${usage}"
```
Comment (edburns) [inline weblogic-azure-aks/src/main/arm/scripts/setupDBConnections.sh:28]: While I support the idea of no longer reading a very long list of parameters from stdin, and instead using environment variables, I do not support losing the helpful usage message.  Please restore the usage message and make it clear what environment variables this script requires to function correctly.

File: weblogic-azure-aks/src/main/bicep/modules/_deployment-scripts/_ds-datasource-connection.bicep [hunk 2]
```diff
@@ -43,8 +41,35 @@ resource deploymentScript 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
   identity: identity
   properties: {
     azCliVersion: azCliVersion
-    arguments: const_arguments
     environmentVariables: [
+      {
+        name: 'AKS_RESOURCE_GROUP_NAME'
+        value: aksClusterRGName
+      }
+      {
```
Comment (edburns) [inline weblogic-azure-aks/src/main/bicep/modules/_deployment-scripts/_ds-datasource-connection.bicep:49]: Yes, this is much cleaner.  I have filed a [User Story](https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1590695) to apply this approach throughout the offer.

**Review by edburns — CHANGES_REQUESTED** [review]
One cosmetic change requested, regarding the `usage` function.

**Review by edburns — APPROVED** [review]

**Review by mriccell — APPROVED** [review]

**Pull request merged**